src/app/features/jobs/components/job-form/job-form.component.html

```diff
@@ -0,0 +1,57 @@
+@if (contractTypes$ | async; as contractTypes) {
```

Comment:
Since this way with the @if is absolutely horrible, let's do it with the *ngIf : 

```
<ng-container *ngIf="{
  contractTypes: contractTypes$ | async,
  jobFunctions: jobFunctions$ | async,
  companyAdvertiser: companyAdvertiser$ | async
} as data">
  <ng-container *ngIf="data.contractTypes && data.jobFunctions && data.companyAdvertiser">
    ...
  </ng-container>
```